tests/test_table.py

```diff
@@ -141,19 +140,19 @@ def test_access():
                 create_table(
                     pd.Series(
                         index=audformat.filewise_index(),
-                        dtype=float,
+                        dtype='object',
                     )
                 ),
                 create_table(
                     pd.Series(
-                        [1.],
+                        ['a'],
```

Comment:
Since we cannot create a `Table` with `dtype='float'` without assigning it to a scheme I changed to `'object'` to make the test pass. The actual dtype is not relevant for the test since we only want to show that an empty `Table` can be combined with a non-empty `Table`. So should be ok.